R/utility.R

```diff
@@ -725,4 +724,117 @@ viewSplines <- function(knots, degree, theta) {
   }
 
   return(p)
-}
\ No newline at end of file
+}
+
+#' Get survival curve parameters
+#'
+#' @param points A list of two-element vectors specifying the desired time and 
```

Comment:
Is there a usability reason why you require a (for R rather unusual) list of two element vectors instead of a more "natural"  data.frame or matrix? (same for the plot function ofc)

Reply:
I did go back and forth on this, but I kind of liked the explicit nature of entering in the pair together. Since this is just a utility function that is really meant to be used on the fly (though it could be part of reproducible code as well), it seemed to make sense. Also, in most cases, folks will only be entering two points, maybe three, so it seemed easy enough. I don't mind being a little different. So, for now, let's keep as is.